---
layout: page
lang-ref: OToL
title: OToL Phylogeny  Working Group
background: /assets/images/circle_phylo.jpg
description: Curating a community consensus phylogney of the legumes
height: 70vh
---


Coordinators: [Joe Miller](mailto:jmiller@gbif.org) [(Global Biodiversity Information Facility) Copenhagen Denmark](https://www.gbif.org) and [Vanessa Terra](mailto:vanessaterrab@gmail.com) Universidade Federal de Uberlândia - UFU

## Introduction

[Open Tree of Life](https://tree.opentreeoflife.org/about/open-tree-of-life) aims to construct a comprehensive, dynamic and digitally-available tree of life by synthesizing published phylogenetic trees along with taxonomic data. I think of it as the phylogenetic version of GBIF. GBIF combines data from thousands of herbarium datasets into a system that allows anyone to access occurrence data in a geographic area or of a taxonomic group.  Likewise OToL, since we don’t have a single analyzed phylogenetic tree of all life does the same with open access data. OToL users can download a subset tree for example for all of Fabaceae or from a list of species.   As with GBIF the better the input data the better the output data.  OToL trees can be used for many purposes including research and for visualizations.


It is important to recognize that the OToL synthetic tree includes terminals from published phylogenetic analyses from many  sources (various genes, sampling and morphology) and terminals that are placed there only by taxonomy.  OToL has a robust Taxonomic Name Resolution Service (TNRS).  Many terminals in the tree are known only from the taxonomy, that is they are not currently represented in a source phylogeny currently in OToL.   For example the Fabaceae clade ([OTT: 560323](https://tree.opentreeoflife.org/opentree/argus/ottol@560323/Fabaceae)) includes 24,479 species but only [4,835 species’](https://docs.google.com/spreadsheets/d/1YQz6F-DOdCTZneGvEfnCtMZs6nyUTmp0N7le86aYnoY/edit?usp=sharing) placement are based on input phylogenies. Species that are not in any input trees are placed as a polytomy at the genus level resulting in polytomies. Therefore we need to input as many high quality trees as possible. With the help of OToL one can download a tree that only includes terminals that are from phylogenetic analyses. For info about OToL see the about tab [here](https://tree.opentreeoflife.org/about/open-tree-of-life).

We have been working closely with [Emily Jane McTavish](mailto:ejmctavish@ucmerced.edu) from UC Merced and a OToL PI. She has sent me much information including the synthetic trees and lists below.  She is keen to work with us to improve the Fabaceae clade of OToL. Please note that it is very easy to import a phylogeny from [Treebase](https://www.treebase.org/treebase-web/home.html) into OToL for further curation.  Therefore harvesting legume [Treebase](https://www.treebase.org/treebase-web/home.html) studies is the first place to improve the OToL synthetic tree.

## Status of Legumes in OToL

| ToL Derived Phylogeny (version Dec 2019)| # of Terminals | link to file                                                                                  |
| --------------------------------------- |:--------------:| ---------------------------------------------------------------------------------------------:|
| All Fabaceae species                    | 24,479         | [at OToL](https://tree.opentreeoflife.org/opentree/argus/ottol@560323/Fabaceae)         |
| Only based on phylogenies      | 4,835          | [Newick](https://drive.google.com/file/d/1OcTQbFTuO8Heo_xAvgbc6XgBAODheWIU/view?usp=sharing)  |

--------

[Published studies currently integrated into current OTol verions](https://drive.google.com/file/d/1KUvDkieslHQF1d_S9tJO0WHRZyelsnTi/view?usp=sharing)

[Partially curated studies in OToL that contain Fabaceae species](https://drive.google.com/file/d/1KOvDi_91SLNrRDDLjKkyRf80COjxs202/view?usp=sharing)

Given the trees that are in the OToL and the distribution of the people interested in the project it may be the best option to start with this subfamily subset: Caesalpinioideae (incl. mimosoid clade), Dialioideae, Detarioideae, Cercidoideae, and Duparquetioideae.
 

Here is what we have as of late January 2021:



This is a [google spreadsheet](https://docs.google.com/spreadsheets/d/1Fvf6UJ7Q35Mu9bmmx4f2gtj7ple2N9OEEIhM4SNnuqA/edit#gid=1233710896) that lists the status of trees in OToL and some potential target trees to import:

![alt text](/assets/images/OToL List.png)

| Tab                   | Status                         | Action                                                                                              |
| --------------------- |:------------------------------:| ---------------------------------------------------------------------------------------------------:|
| OToL List             | Completed                      | Already in OToL, no action needed                                                                   |
| OToL List             | In Progress                    | Considering your knowledge of the clade and paper, is this important to include?                    |
| OToL List             | Priority                       | Update Priority in column 2                                                                         |
| Potential Trees       | In Treebase                    | Import into OToL from TreeBase and curate                                                           |
| Potential Trees       | pdf, not in treebase           | Determine if priority, find tree or contact authors or are there newer studies that cover this clade|
| Potential Trees       | to assess                      | Is this worth chasing? or are there newer studies that cover this clade                             |
| Potential Trees       | find pdf                       | Is this worth chasing? or are there newer studies that cover this clade                             |
| Suggest a tree        | Trees needed                   | Recent trees that cover groups not in current synthetic treed, especially species level             |

--------
* Potential Trees: This list of trees was pulled mailnly from LPWG citations. This is an old list and needs updating.
 
## Priority list of things to do by the Working group
* Identify studies that are high priority to include
   * species level
   * missing in current synthetic tree
   * _recent publications_
   * trees that resolve the backbone, new Next Gen studies
   * in Treebase or we can easily get Treefiles
* Identify trees from the OToL List and Potential Tree list that are worthy of incusion
* Download the current OToL tree file [Newick](https://drive.google.com/file/d/1OcTQbFTuO8Heo_xAvgbc6XgBAODheWIU/view?usp=sharing) and find gaps in coverage
* Update the taxonomy in the spreadsheet
   * to help find gaps
   * many mistakes
* Perhps focus on Caesalpinioideae (incl. mimosoid clade), Dialioideae, Detarioideae, Cercidoideae, and Duparquetioideae
   * much better current coverage in these clades


## How curate files in OToL
Click on a OpenTree study link to see what the interface looks like.  You will need a GitHb account to edit. More details to be added later. The OToL syntehitc tree is only updated occasionaly. The most recent was in December 2019. Updating the legume clade would be good incentive for a new synthetic tree update by OToL.


## Publication idea
We could rewrite a paper describing the before and after this project. How many species and studies did we add? How much better/useful is the OToL phylogeny? Provide examples of how the tree would be used, especially in connection to the taxonomic georeferencing group work.  GBIF is working on a phylogenetic viewer that could be added to the Legume Data Portal.

## Join the Working Group
Please send an email to [Joe Miller](mailto:jmiller@gbif.org) or [Vanessa Terra](mailto:vanessaterrab@gmail.com) or just start looking for studies to include.  No experience in curating OToL files is necessary.  The most important  thing we need is your knowledge of the best trees to include.



![alt text](/assets/images/circle_phylo.webp)
